terraform/modules/database/database.tf

```diff
@@ -9,6 +9,13 @@ resource "aws_rds_cluster_instance" "cluster_instances" {
   auto_minor_version_upgrade = "true"
   performance_insights_enabled = "true"
   preferred_maintenance_window = "${var.preferred_maintenance_window}"
+  tags = "${merge(
+    var.default_tags,
+    map(
+      "Name","upload",
```

Comment:
Use a meaningful name here like `upload-srv-${var.deployment_stage}-rds-cluster`

Reply:
Actually, I was thinking if do we really need name tags? I believe each resources will already have its name property. How will that be useful? Is this in the compliance rules?